contributor-guide/modules/ROOT/pages/requirements/organization-requirements.adoc

```diff
@@ -71,23 +73,26 @@ run cpp11_test.cpp ;
 
 For more information, see the documentation of Boost.Config.
 
+[#building_documentation]
 === Building Documentation
 
 The library needs to provide a Boost Build project for building the documentation for the library. The project-root/doc project is the only location referred to by the top level documentation build scripts and the release building scripts. The documentation build project must have the following two features:
 
-Define a boostdoc target. This target should likely be an alias that looks roughly like:
+Define a `boostrelease`  target. This target should likely be an alias that looks roughly like:
 
 [source,bash]
 ----
-alias boostdoc : my_boostbook_target
+alias boostrelease  : my_boostbook_target
     : : : <implicit-dependency>my_boostbook_target ;
 ----
 
 But if your project doesn't integrate into the global documentation book you can use an empty alias like:
 
 [source,bash]
 ----
-alias boostdoc ;
+alias boostrelease  ;
 ----
 
-The project must default to building standalone documentation if it has any. The release scripts build this default so as to guarantee all projects have up to date documentation.
\ No newline at end of file
+The project must default to building standalone documentation if it has any. The release scripts build this default so as to guarantee all projects have up to date documentation.
+
+Note:: _Integrated_ documentation, using the `boostdoc` target (instead of the `boostrelease` target) is now considered legacy, and should be avoided for new library documentation.
```

Comment:
Also useful information. People ask about this all the time. At some point, we might be able to move all the documentation to antora and this won't be a problem.